api/src/main/java/io/minio/MinioClient.java

```diff
@@ -4254,925 +3951,39 @@ public void removeBucket(String bucketName)
   }
 
   /**
-   * Uploads given file as object in given bucket.
-   * <p>
-   * If the object is larger than 5MB, the client will automatically use a multipart session.
-   * </p>
-   * <p>
-   * If the session fails, the user may attempt to re-upload the object by attempting to create
-   * the exact same object again.
-   * </p>
-   * <p>
-   * If the multipart session fails, we abort all the uploaded content.
-   * </p>
-   *
-   * @param bucketName  Bucket name.
-   * @param objectName  Object name to create in the bucket.
-   * @param fileName    File name to upload.
-   *
-   * @throws InvalidBucketNameException  upon invalid bucket name is given
-   * @throws NoSuchAlgorithmException
-   *           upon requested algorithm was not found during signature calculation
-   * @throws IOException                 upon connection error
-   * @throws InvalidKeyException
-   *           upon an invalid access key or secret key
-   * @throws XmlPullParserException      upon parsing response xml
-   * @throws ErrorResponseException      upon unsuccessful execution
-   * @throws InternalException           upon internal library error
-   * @throws IllegalArgumentException    upon invalid value is passed to a method.
-   * @throws InsufficientDataException   upon getting EOFException while reading given
-   * @throws InvalidResponseException    upon a non-xml response from server
+   * Executes put object and returns ETag of the object.
    *
-   * @deprecated As of release 7.0
+   * @param bucketName
+   *          Bucket name.
+   * @param objectName
+   *          Object name in the bucket.
+   * @param length
+   *          Length of object data.
+   * @param data
+   *          Object data.
+   * @param uploadId
+   *          Upload ID of multipart put object.
+   * @param partNumber
+   *          Part number of multipart put object.
    */
```

Comment:
we should keep the comment about exceptions being thrown to be consistent

Reply:
Its odd to have javadoc for private methods. I would suggest to remove such unnecessary docs.

Reply:
ok. got it.